src/diem/testing/local_account.py

```diff
@@ -146,6 +146,22 @@ def rotate_dual_attestation_info(
             stdlib.encode_rotate_dual_attestation_info_script(new_url=base_url.encode("utf-8"), new_key=compliance_key),
         )
 
+    def gen_child_vasp(self, client: jsonrpc.Client, initial_balance: int, currency: str) -> "LocalAccount":
+        """Generates a new ChildVASP account if `self` is a ParentVASP account."""
```

Comment:
out of curiosity, what happens if it's not? transaction fail?

Reply:
transaction execution will fail

Reply:
updated comment